ffi/capi/src/exemplar_chars.rs

```diff
@@ -69,7 +83,21 @@ pub mod ffi {
             FnInStruct
         )]
         #[diplomat::attr(supports = fallible_constructors, named_constructor = "auxiliary")]
-        pub fn try_new_auxiliary(
+        #[cfg(feature = "compiled_data")]
+        pub fn try_new_auxiliary(locale: &Locale) -> Result<Box<ExemplarCharacters>, DataError> {
```

Comment:
"using compiled data"

And elsewhere in this file.

Also, should these be called "create" instead of "try_new"?

Reply:
Yes, they should.